content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+# Announcing scip-clang: a new SCIP indexer for C and C++
+
+[scip-clang](https://github.com/sourcegraph/scip-clang) is our new indexer
+for C and C++ code written from the ground up to natively emit [SCIP](https://github.com/sourcegraph/scip)
+and especially support the wide range of language features present in C++.
```

Comment:
I feel like there are two personas we ought to target in this announcement, existing customers who are familiar with other scip-indexers or lsif-clang, they're interested in support/robustness/new features.  The second persona are customers that are *not* using lsif-clang or are prospects, what makes scip-clang (and specifically precise intelligence for C++) impressive? What do they gain over a heuristic approach (either ours or competitors)?

Reply:
Would you like me to spell that out right at the beginning? Maybe I should fit some points in below the demo?

Reply:
I think perhaps earlier would be best and then it can roll into the demo.  I'll leave it up to you to find where it feels more natural.

Reply:
Can you take a second look here? I've added a paragraph with some bullet points

Reply:
Looks good.